build_projects/dotnet-cli-build/PublishTargets.cs

```diff
@@ -47,9 +48,9 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
         {
             if (CheckIfAllBuildsHavePublished())
             {
-                string targetContainer = $"{Channel}/Binaries/Latest/";
+                string targetContainer = $"{AzurePublisher.Product.Sdk}/{Channel}/";
                 string targetVersionFile = $"{targetContainer}{CliNuGetVersion}";
-                string semaphoreBlob = $"{Channel}/Binaries/publishSemaphore";
+                string semaphoreBlob = $"{targetContainer}/publishSemaphore";
                 AzurePublisherTool.CreateBlobIfNotExists(semaphoreBlob);
                 string leaseId = AzurePublisherTool.AcquireLeaseOnBlob(semaphoreBlob);
```

Comment:
Alternatively we can do the same as in core-setup and just wait.

---
In reply to: [67969277](https://github.com/dotnet/cli/pull/3669#discussion_r67969277) [](ancestors = 67969277,67969144)